src/state/ol3/reducers.js

```diff
@@ -0,0 +1,19 @@
+import {GET_OL3_MAP, SET_OL3_MAP} from '../actiontypes';
+
+const defaultState = {
+  ol3Map: undefined
```

Comment:
in general, openlayers is moving away from version numbers in names, so olMap might be better?